src/java.desktop/share/classes/sun/font/CreatedFontTracker.java

```diff
@@ -112,25 +112,20 @@ private static class TempFileDeletionHook {
         private static HashMap<File, OutputStream> files = new HashMap<>();
 
         private static Thread t = null;
-        @SuppressWarnings("removal")
```

Comment:
This whole file may be obsolete now. But I stuck to the doPrivileged brief, and updated the existing tracker clean up bug to include looking at this file : https://bugs.openjdk.org/browse/JDK-8344146